packages/sources/enzyme/test/unit/calcgav.test.ts

```diff
@@ -0,0 +1,45 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src'
+import { ENV_RPC_URL } from '../../../curve/src/config'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+  process.env[ENV_RPC_URL] = process.env[ENV_RPC_URL] || 'http://localhost:8546/'
```

Comment:
Not important but you can also rewrite this as

```js
process.env.ENV_RPC_URL = process.env.ENV_RPC_URL || 'http://localhost:8546/'
```

to keep it consistent with the other tests.

Reply:
The env var we're reading is actually `RPC_URL`, doing `process.env[ENV_RPC_URL]` is just to allow us to reference a constant (`ENV_RPC_URL`) to make sure that this remains the same throughout the entire adapter.

Reply:
I see. Thanks for clarifying :+1:

This is just a nitpick, but it caught my eye during review so I thought that I just add a review comment 😅